NebulaPatcher/Patches/Transpilers/PlanetFactory_Patch.cs

```diff
@@ -43,5 +43,58 @@ static IEnumerable<CodeInstruction> DestructFinally_Transpiler(ILGenerator gen,
             }
             return codes;
         }
+
+        [HarmonyTranspiler]
+        [HarmonyPatch("OnBeltBuilt")]
+        static IEnumerable<CodeInstruction> OnBeltBuilt_Transpiler(ILGenerator gen, IEnumerable<CodeInstruction> instructions)
+        {
+            var codes = new List<CodeInstruction>(instructions);
+            for (int i = 0; i < codes.Count; i++)
+            {
+                if (codes[i].opcode == OpCodes.Callvirt && codes[i].operand.ToString() == "Void SetInserterPickTarget(Int32, Int32, Int32)" &&
```

Comment:
you could probably also do something like `((MethodInfo)codes[i].operand).Name == "SetInserterPickTarget"` but i guess thats just personal preference :D

Reply:
I think I would prefer this suggestion, just in case an arg changes or something

Reply:
fixed :)